java/com/google/turbine/options/TurbineOptionsParser.java

```diff
@@ -175,6 +174,7 @@ private static void expandParamsFiles(Deque<String> argumentDeque, Iterable<Stri
       if (arg.charAt(0) == '\'') {
         // perform best-effort unescaping as a concession to ninja, see:
         // https://android.googlesource.com/platform/external/ninja/+/6f903faaf5488dc052ffc4e3e0b12757b426e088/src/util.cc#274
+    	// FIXME: do we need this? OptionParser in Bazel's javac compiler does not support this at all!
```

Comment:
@cushon Do you have an idea why this escaping was added to Turbine only but not in Bazel?

Reply:
See the comment above--it was added to support a use of turbine with the Ninja build system